js/classes/entities/midground/enemies/wizard.js

```diff
@@ -0,0 +1,118 @@
+class Wizard {
+   /**
+     * @param {Vector} pos the position at which he should spawn. 
+     * @param {Conversation} convo The conversation that will show if BlackSmith is interacted with.
+     */
+    constructor(pos, convo = null) {
+        /** The position of the Papa Chad (in the game world). */
```

Comment:
You're telling me that Papa Chad and the wizard are the same person??

Reply:
lotta copy and paste lmao.